### PR TITLE
Handle uncatched unicode decoding errors

### PR DESCRIPTION
**Pull request by m-schmoock**

### CODE REVIEW

File: electrumpersonalserver/server/common.py [hunk 1]
```diff
@@ -132,11 +133,11 @@ def send_reply_fun(reply):
                         line = recv_buffer[:lb].rstrip()
                         recv_buffer = recv_buffer[lb + 1:]
                         lb = recv_buffer.find(b'\n')
-                        line = line.decode("utf-8")
-                        logger.debug("=> " + line)
                         try:
+                            line = line.decode("utf-8")
+                            logger.debug("=> " + line)
```
Comment (chris-belcher) [inline electrumpersonalserver/server/common.py]: Move this line outside the try/except block, to limit the scope of the exceptions to as little code as possible.

**Comment by chris-belcher** [comment]
In the git commit message, please remove "fix: " and capitalize the first letter of catch.

See also https://chris.beams.io/posts/git-commit/#seven-rules

**Comment by m-schmoock** [comment]
@chris-belcher like this?

**Comment by chris-belcher** [comment]
Yes. Although now there's a grammar mistake, it should be "uncaught" not "uncatched".

**Comment by m-schmoock** [comment]
alrighty then...

File: electrumpersonalserver/server/common.py (hunk 1 above)
Comment (m-schmoock) [reply electrumpersonalserver/server/common.py]: sorry, I missed that. The debug line is now after the try/except block.

**Comment by m-schmoock** [comment]
@chris-belcher can it be merged?

**Comment by chris-belcher** [comment]
I was waiting for the grammatical error to be fixed, but it's not that important.

**Comment by m-schmoock** [comment]
Whoops, I thought I did this though

**Pull request merged**